learning-bitmark/working-with-bitmarks/payment-for-bitmark-cli.md

```diff
@@ -0,0 +1,261 @@
+# Bitmark transaction payment
+
+In the Bitmark Property System, for each asset, each Bitmark Account is able to issue a bitmark for free, and has to pay for all the other transactions such as bitmark transfers or other bitmark issuances.
+
+The Bitmark CLI submits transactions to its connected node via RPC. Once receiving these transactions, the node processes them and returned the corresponding payment ids along with the payment amounts and addresses. Only when the payment - either by bitcoin or litecoin - is executed, a transaction is updated from `Pending` to `Verified` and is ready to be mined. The payment transaction needs to send:
+
+* A correct amount
+
+* To the correct address
+
+* With the correct payment id added as bitcoin/litecoin OP_RETURN opcode.
+
+Because common wallets do not support attaching the Bitmark blockchain payment identifier to the ltc/btc transactions, we provide an app called the Bitmark Wallet to pay for the Bitmark transactions come from the Bitmark CLI.
+
+<br>
+<br>
+
+## Bitmark Wallet
+
+The Bitmark Wallet is a dual currency wallet, supporting both bitcoin and litecoin. It allows sending multiple payments to different addresses and attaching a hex data item to a transaction using the OP_RETURN feature of bitcoin/litecoin.
+
+The wallet is written in Go and cannot work standalone, it can only send the bitcoin and litecoin transactions via a correct configured bitcoind and litecoind. Depending on the configuration of bitcoind and litecoind, the payment transactions are submitted to the mainnet coins or testnet coins.
+
+Supported commands:
+
+* Submit btc/ltc transactions.
+
+* Update the wallet from the btc/ltc block chain - for this it will query its local daemon to request any transctions belonging to its addresses.
+
+* Display the wallet balance for each coin.
+
+* Generate and display an address that can be used to send more coins to the wallet.
+
+> **NOTE:**The Bitmark Wallet uses a master bip39 based recovery phrase which must be printed and kept safe.
+
+<br>
+
+Next steps:
+
+* Install the Bitmark Wallet.
+
+* Sync the wallet with the coins.
+
+* Run the wallet.
+
+* Get the testnet coins.
+
+<br>
+<br>
+
+### Prerequisites of installing the Bitmark Wallet
+
+* Git has been installed.
+
+* Go has been installed.
+
+* Python3 is installed.
+
+* bitcoind/litecoind have been installed.
+
+<br>
+<br>
+
+### Installing the Bitmark Wallet
+
+* Clone the Bitmark Wallet
+
+    ```shell
+        $git clone https://github.com/bitmark-inc/bitmark-wallet.git
+    ```
+
+<br>
+
+* Install the Bitmark Wallet
+
+    ```shell
+        $cd bitmark-wallet/command/bitmark-wallet
+
+        $go install
+    ```
+
+<br>
+
+* Create Bitmark Wallet's config file
+
+    ```shell
+        $ touch <CONFIG DIR>/bitmark-wallet/bitmark-wallet.conf
+    ```
+
+    *Example:*
+    ```shell
+        $ mkdir ${HOME}/.config/bitmark-wallet
+        $ touch ${HOME}/.config/bitmark-wallet/bitmark-wallet.conf
+    ```
```

Comment:
Which is more common, "\~" or "${HOME}"? I tend to use "~" in tutorials like this, because I think all the standard shells use it.

Reply:
Updated.